scripts/parse_sample_file_map.py

```diff
@@ -80,6 +85,7 @@ async def main(
         default_sequence_type=default_sequence_type,
         search_locations=search_path,
         allow_extra_files_in_search_path=allow_extra_files_in_search_path,
+        default_reference_assembly_location=ref
```

Comment:
Linter wants a comma here

```suggestion
        default_reference_assembly_location=ref,
```

Reply:
Cheers. I need to sort out my black settings.